### DISCUSSION TITLE
Error 500 (HTTP ERROR) when logging in to the site

### DISCUSSION THREAD

**Discussion by SHAWNiksDev** [mastodon/mastodon#28284] [General]
### Steps to reproduce the problem

![image translated](https://github.com/mastodon/mastodon/assets/138322362/2ff82e25-95c2-49c7-b0ef-c7f440c9bd20)

### Expected behaviour

To load the site

### Actual behaviour

The site did not load with error 500

### Detailed description

Hello everyone I followed the instructions on this site: https://docs.joinmastodon.org/admin/prerequisites/ 
But after installation, the Mastodon page does not open for me. Error 500 pops up

### Mastodon instance

clipfun.ru

### Mastodon version

4.1.2

### Browser name and version

YandexBrowser 23.11.0.2470 (64-bit)

### Operating system

Ubuntu 20.04.6

**Comment by StrangeMinds1960** [comment]
I assume https://clipfun.ru is the site you're trying to load?  Works fine for me, you might want to try a different browser

**Reply by SHAWNiksDev** [reply]
It's okay! I have solved this problem. When installing, I was asked: "Are you going to run Mastodon in docker?" I had to answer "No" and after setting up the database, the server started working
